news/2023/2023-06-20-beach-episode-art-contest.md

```diff
@@ -14,26 +14,26 @@ It's been hot as hell this year, so what better way to welcome things in than wi
 
 **Draw any of the official or community mascots having fun at a beach!**
 
-That's it! If it's an activity someone can, might, or reasonably has at a beach before, it's fair game if you draw any one of our mascots doing them.
+That's it! If it's an activity someone can, might, or reasonably has had at a beach before, it's fair game if you draw any one of our mascots doing them.
```

Comment:
"an activity someone has at the beach" seems like correct english to me but who am i to say

Reply:
notice "an activity someone has at the beach **before**" is wrong.

you're right that it's wonky because the tenses don't match all three items ("can have ~~before~~", "might have ~~before~~", "reasonably has had before"), can reword though:

```suggestion
That's it! If it's an activity someone could, might, or reasonably has had at a beach before, it's fair game if you draw any one of our mascots doing them.
```

or if you want present tense:

```suggestion
That's it! If it's an activity someone can, might, or reasonably has at a beach, it's fair game if you draw any one of our mascots doing them.
```

Reply:
let's leave this as you have it i think it's ok